tests/kafkatest/services/security/security_config.py

```diff
@@ -19,6 +19,7 @@
 from shutil import rmtree
 from ducktape.template import TemplateRenderer
 from kafkatest.services.security.minikdc import MiniKdc
+from kafkatest.services.kafka.listener_config import ListenerConfig
```

Comment:
This introduces a circular dependency. We should move `listener_config.py` to `tests/kafkatest/services/security` so that SecurityConfig doesn't depend on Kafka.